### ISSUE TITLE
bbl director-address prints out migration messages to stdout

### ISSUE THREAD

**Issue by dsabeti** [cloudfoundry/bosh-bootloader#67]
It looks like bbl is printing out a migration message because `state.json` is changing to `bbl-state.json`:

```
$ bbl director-address
renaming state.json to bbl-state.json
https://52.44.157.91:25555
```

The issue is that the migration message is printed to STDOUT. So, if you have a script that depends on the output of `bbl director-address` -- for example, [here](https://github.com/cloudfoundry/runtime-ci/blob/master/scripts/ci/extract-director-creds-from-bbl/task#L3) -- then the script may break or have unexpected behavior as a result of the migration message.

Could it move to stderr instead? Or maybe the command should just fail with a message like "State.json needs to be migrated. Please run `bbl up` first to migrate..."

**Comment by Amit-PivotalLabs** [comment]
Thanks @dsabeti 

I think we should make it so that:

(a) only `bbl up` does the migration
(b) other commands which detect a `state.json` but no `bbl-state.json` should fail fast, telling you to run `bbl up` first and exiting 1 (even if flags like `--skip-if-missing` are passed).
(c) other commands which detect both files fail fast, exiting 1, complaining that both files are found in the state directory.
(d) other commands which detect neither file in the state dir maintain their existing behaviour when not finding the state file.

**Comment by zankich** [comment]
This has been fixed in bbl [1.2](https://github.com/cloudfoundry/bosh-bootloader/releases/tag/v1.2.0)
